Documentation/Signoffs/Microcontroller.md

```diff
@@ -0,0 +1,44 @@
+# Microcontroller Signoff
+## FUNCTION OF THE SUBSYSTEM
+Within this subsystem, the code will instruct the hardware in how to sample from the sensors, send data to the Communication subsystem, format and store data onto a USB Flash drive, and monitor battery voltage and current levels from the power subsystem.  These are the core functions that are desired from the microcontroller.  
+
+## CONSTRAINTS ON THE SUBSYSTEM
+The constraints given below are generalized to allow for comparison of any potential contender for the microcontroller.  Constraints on the subsystem are as follows:
+1. The chosen microcontroller should cost no more than $60 as the end-user only has a budget of $300.  This board value cap is gathered from estimating the costs of other subsystems and viewing prices of available options that satisfy proceeding constraints such as Raspberry Pis, Arduinos, Intel, ESP, etc.  Up to 20% of the budget was then allocated due to the importance of this subsystem.
+2. The microcontroller must be able to send instructions and transfer data to other components such as the Communication subsystem.  To do this, Serial Peripheral Interface (SPI) or Universal Asynchronous Receiver/Transmitter (UART) (2 extra UART pins) as standards in hardware communication for liaising will need to be achievable. 
```

Comment:
Which one, SPI or UART?

Reply:
We did not want to limit ourselves as both would work.   We will choose UART if sticking to one is preferred.